theme/scripts/utils/jsonp.ts

```diff
@@ -0,0 +1,53 @@
+let count = 0;
+
+const noop = () => {}
+
+const jsonp = (url, param, fn) => {
+  var prefix = '__jp';
+
```

Comment:
I see this is "cleaning up" timers and removing scripts but not sure why?

Reply:
`jsonp` is a handy way to make a server request, but bypassing CORS policies. This SO thread sort of explains it: https://stackoverflow.com/questions/2067472/what-is-jsonp-and-why-was-it-created